install.sh

```diff
@@ -15,33 +13,50 @@ apt-get --yes install epoptes epoptes-client
 
 
 #Adding vagrant user to group epoptes
-
 gpasswd -a ${SUDO_USER:-$USER} epoptes
 
 #Updating kernel
-
 echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
 #configure dnsmasq
-
 ltsp-config dnsmasq
 
 #Creating lts.conf
-
 ltsp-config lts.conf
 
 #Installing additional software
-
 apt-get --yes install edubuntu-desktop
 apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
 #Installing ltsp-manager
-
 add-apt-repository ppa:ts.sch.gr
 apt-get --yes update
 apt-get --yes install ltsp-manager
 
 #Creating client image
-
 ltsp-update-image --cleanup /
+
+#source setting.sh
+source /vagrant/settings.sh
+
+#setting dhcp-range
+sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf -e "/8h$/ c dhcp-range=${NETWORK}.20,${NETWORK}.250,8h"
+
+#Setting mode of operation of ltsp server
+if [[ ${STANDALONE,,} == "yes" ]]; then
+	echo "LTSP server will be in standalone mode of operation"	
+	echo "LTSP server will provide DHCP services.."	
+	sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf -e "/192.168.1.0,proxy$/ c #dhcp-range=${NETWORK}.0,proxy"
+	sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf -e "/10.0.2.0,proxy$/ c #dhcp-range=10.0.2.0,proxy"
+elif [[ ${STANDALONE,,} == "no" ]]; then
```

Comment:
Make this just `else`. So the default will be non-standalone for any value different from `yes` or `Yes` or `YES`.